bool gesture0
bool gesture1
bool gesture2
bool gesture3
bool gesture4
bool gesture5
bool gesture6
bool gesture7
bool gesture8
bool gesture9
bool gesture10
bool gesture11
bool gesture12
bool gesture13
bool gesture14
bool gesture15
bool gesture16
bool gesture17
bool gesture18
bool gesture19


